docs/content/docs/integrations/artifact-manager.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Artifactory
+main_menu: true
+card: 
+  name: artifact-manager
+---
+
+The Artifactory integration used the builtin integration model "Artifact Manager" and can be configured on every project by users
+
+This integration allows you:
+
+* to upload/download artifact inside artifactory
+* create a new artifactory build from your workflow run results
+* create a release bundle using artifacts previously uploaded in artifactory
+
+## Recommendations
+
+To make full use of this integration, we recommend a few things
+
+* Naming your local repositories like this: [team]-[technology]-[maturity]
+* Having virtual repositories named like this: [team]-[technology]
+
+For example, you need a generic repository for CDS artifacts. You can create something like:
+
+* Virtual repository: myteam-docker
+* Snapshot repository: myteam-docker-snapshot
+* Release repository: myteam-docker-release
+
+By using this convention, the CDS release action will be able to promote your artifacts from snapshot to release repositories
+
+## How to configure Artifactory integration on your project
+
+### Link Artifactory to your project
+
+On the integration project view, add a new "Artifact Manager" integration and fill the following parameters:
+
+* Name: The name of the integration.
+* platform: Must be 'artifactory'
+* url: URL of artifactory api (https//myinstance.ofartifactory/artifactory/)
+* project.key: The name of the artifactory project (https://www.jfrog.com/confluence/display/JFROG/Projects)
+* cds.repository: The name of the repository used by CDS to upload/download artifacts (must be a virtual repository)
+* token.name: The name of the access token used by CDS to access the artifactory API
+* token: The value of the access token used by CDS to access the artifactory API
+* release.token: The value of the access token used by CDS to access the distribution API (https://www.jfrog.com/confluence/display/JFROG/JFrog+Distribution)
+* promotion.maturity.low: suffix used on your local repositories to identify your snapshots
+* promotion.maturity.high: suffix used on your local repositories to identify your releases
+
+### Enable Artifactory integration on your workflow
+
+On the workflow advanced view, you can link your workflow to project integration.
+
+Select the artifactory integration and fill the parameter:
+
+* build.info.path: prefix used by CDS action Push Build Info to name your build info (see below for more detailed informations). For example it can be the name of your team.
```

Comment:
```suggestion
* `build.info.path`: prefix used by CDS action Push Build Info to name your build info (see below for more detailed informations). For example it can be the name of your team.
```